### ISSUE TITLE
Wrong links to demos

### ISSUE THREAD

**Issue by lrineau** [CGAL/cgal#3888]
I ran the link checker over the current head of the 4.14 branch (i.e. 4.14.1): and have some problems with:
```
https://www.cgal.org/demo/4.14.1/boolean_operations_2.zip
https://www.cgal.org/demo/4.14.1/envelope_3.zip
https://www.cgal.org/demo/4.14.1/boolean_operations_2.zip
https://www.cgal.org/demo/4.14.1/partition_2.zip
https://www.cgal.org/demo/4.14.1/straight_skeleton_2.zip
https://www.cgal.org/demo/4.14.1/envelope_3.zip
https://www.cgal.org/demo/4.14.1/nef_2.zip
https://www.cgal.org/demo/4.14.1/nef_2.zip
https://www.cgal.org/demo/4.14.1/partition_2.zip
https://www.cgal.org/demo/4.14.1/straight_skeleton_2.zip
```
The ones I tested give a message like:

> 
> The page isn’t redirecting properly
> 
> An error occurred during a connection to www.cgal.org. 

and is redirecting to e.g. `https://www.cgal.org/demo/3.5/nef_2.zip`

_Originally posted by @albert-github in https://github.com/CGAL/cgal/issues/3817#issuecomment-484571902_

**Comment by lrineau** [comment]
All those demos are old Qt3 demos, that no longer exist for a long long time ago! We should fix or remove the links in the documentation.

**Comment by lrineau** [comment]
Fixed by #3889.
